modules/apps/transfer/types/trace.go

```diff
@@ -70,6 +74,22 @@ func (dt DenomTrace) GetFullDenomPath() string {
 	return dt.GetPrefix() + dt.BaseDenom
 }
 
+func extractPathAndBaseDenomFromDenomTrace(denomTraceItems []string) (string, string) {
```

Comment:
would be nice to add a high-level overview of the procedure used to extract the path